unity-renderer/Assets/DCLPlugins/BuilderInWorld/HUD/ProjectsPanelHUD/Tests/ProjectCardViewShould.cs

```diff
@@ -82,49 +85,50 @@ public void SetSyncCorrectly(string date)
         {
             CreateScene()
         };
-
+    
         ((IProjectCardView)cardView).Setup(new ProjectData()
         {
             id = "",
             title = "TesTitle",
             rows = 2,
             cols = 2,
             updated_at = updatedTime,
-            scene_id = sceneList[0].id
+            // scene_number = sceneList[0]
```

Comment:
If it were up to me I'd delete all the BiW code (and tests of course) from the repo but since that is another monumental task, I think it's better to tackle that in a different issue to avoid engrossing this PR.

I left all the existent BiW tests but turned most of them Explicit to avoid running them on the CI (`[Explicit("BIW is deprecated")]`), and those that conflicted with the scene number replacement may have some commented-out code.

After merging this PR I'll create an Issue to remove all the BiW code.

Reply:
ok then!

Reply:
What I'm doing is commeting all the BiW tests when I have to "fix" them. If you do so, dont forget to add an `Obsolete` attribute.